src/main/groovy/com/xebialabs/gradle/xldeploy/XlDeployPluginExtension.groovy

```diff
@@ -41,4 +41,8 @@ class XlDeployPluginExtension {
    */
   String xldEnvironmentId
 
+  /**
+   * Socket timeout for requests to XLD
+   */
+  Integer socketTimeout = 10000
```

Comment:
Can we keep the default value to 1000, similar like the actual BooterConfig default value?

Reply:
I copied this value from the Maven plugin... don't know if it's a good idea to have a default which we already saw is too low. I also don't see any downside of the higher value.

Reply:
Agreed. Just checked the maven plugin code also.